obsplus/utils/stations.py

```diff
@@ -38,15 +49,42 @@ def df_to_inventory(df) -> obspy.Inventory:
     df
         A dataframe which must have the same columns as the once produced by
         :func:`obsplus.stations_to_df`.
```

Comment:
I might recommend actually listing the columns here (network, station, location, channel, seed_id, latitude, longitude, elevation, depth, start_date, end_date, dip, azimuth, sampling_rate (sample_rate?), if you don't have them handy).

I had to actually go convert an inventory to a dataframe to figure out what was required here.